.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [main]
 
 jobs:
-  fast:
+  test:
     strategy:
```

Comment:
Why change name?

Reply:
I think it was named `fast` because it used the `ci-fast` hypothesis profile, which used less hypothesis iterations (10) as opposed to the `ci` hypothesis profile which used 250. It felt appropriate to just make this job a generic test one since the profiles aren't used anymore.